docs/stock/custom-technical-indicators.md

```diff
@@ -328,4 +328,42 @@ Highcharts.seriesType('customIndicator', 'sma', {}, {
 ```
 
 
-A live demo of the example above can be found [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/custom-indicator-on-event/).
\ No newline at end of file
+A live demo of the example above can be found [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/indicators/custom-indicator-on-event/).
+
+
+### 5. Custom column indicator.
+
+The principle of creating a custom column indicator is similar to the ordinary one.
+Still, the `getValues` method for calculating points position is required.
+Along with that, in order to render the column, some other methods like `crispCol`, `drawPoints`, `getColumnMetrics`, `translate`
+need to be inherited from the column series prototype and the `markerAttribs`,  `drawGraph` need to be overwritten with empty function
```

Comment:
Double space before `drawGraph`

Reply:
I'm not good with that, but should it be `need to be overwritten with empty function` or `need to be overwritten with an empty function`?